pontoon/base/templates/widgets/heading_info.html

```diff
@@ -130,3 +130,21 @@
   </div>
 </div>
 {% endmacro %}
+
+{% macro download_button(locale, project) %}
```

Comment:
Sorry for not noticing this earlier. Can we call it `download_selector` instead, to be in line with the class name and the CSS file name?